### PR TITLE
Af4 redrive af3 notes

### PR DESCRIPTION
**Pull request by WhiteAu**

### CODE REVIEW

File: active/operators/resynthesis/resynthesize_notes.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+                    message = "Exception occurred: {}".format(e)
+                    common_functions.log_error_and_failure_for_resynth_note_job(run_id, blobid, hdcpupdatedate, message,
+                                                                      "Save JSON Resynthesis Annotation")
+                    continue
+
+                _update_resynth_run_details_to_complete(run_id, blobid, hdcpupdatedate)
+                record_processed += 1
+
+        _update_job_id_as_complete(run_id)
+        print("{} records processed for update date: {}".format(record_processed, hdcpupdatedate))
```
Comment (esilgard) [inline active/operators/resynthesis/resynthesize_notes.py]: this is essentially a bulk move from the main script out into an operator, yes?

File: active/operators/resynthesis/resynthesize_notes.py [hunk 2]
```diff
@@ -1,9 +1,21 @@
...
+import json
+
+from datetime import datetime, timedelta
+from pymssql import OperationalError
+
+import utilities.common_variables as common_variables
+import utilities.common_hooks as common_hooks
+import utilities.common_functions as common_functions
+
+RESYNTH_JOB_ID = 'resynth_jobs'
```
Comment (esilgard) [inline active/operators/resynthesis/resynthesize_notes.py]: just want to call out - this is slightly different that we're using a common variable across scripts that's called from an operator...I trust your judgement, but maybe we can discuss at stand up tomorrow.

**Review by esilgard — APPROVED** [review]
LGTM - I know you've tested it out - couple things I'd like for us to talk through tomorrow @ standup before it gets merged in

File: active/operators/resynthesis/__init__.py [hunk 3]
```diff
@@ -1 +1,2 @@
-from .fetch_annotations import get_resynth_ready_annotations_since_date
\ No newline at end of file
+from .fetch_annotations import get_resynth_ready_annotations_since_date
+from .resynthesize_notes import resynthesize_notes_marked_as_deid, RESYNTH_JOB_ID
```
Comment (WhiteAu) [inline active/operators/resynthesis/__init__.py]: remove RESYNTH_JOB_ID from this line.

File: active/operators/trashman/__init__.py [hunk 4]
```diff
@@ -1,7 +1,7 @@
 from .generate_job_id import generate_job_id
 from .check_brat_completeness import check_brat_completeness
 from .check_brat_staleness import check_brat_staleness
-from .check_resynth_tasks import check_resynth_tasks
+from .check_resynth_tasks import check_resynth_tasks, REDRIVEABLE_RESYNTH_ID
```
Comment (WhiteAu) [inline active/operators/trashman/__init__.py]: change this back.

File: active/operators/trashman/redrive_jobs.py [hunk 5]
```diff
@@ -21,7 +16,12 @@ def redrive_brat_stale_task(task):
     return
 
 
-def redrive_resynth_jobs(**kwargs):
+def redrive_resynth_jobs(upstream_task, **kwargs):
+    job_tuples =  kwargs['ti'].xcom_pull(task_ids=upstream_task, key=common_variables.REDRIVEABLE_RESYNTH_ID)
+    print("job tuples: {}".format(job_tuples))
```
Comment (WhiteAu) [inline active/operators/trashman/redrive_jobs.py]: remove PHI print stmt

File: active/af4-trashman.py [hunk 6]
```diff
@@ -81,13 +82,20 @@
                                       op_args={'check_resynth_tasks'},
                                       dag=dag)
 
+resynthesize_redriven_notes = \
+    PythonOperator(task_id='resynthesize_redriven_notes',
```
Comment (WhiteAu) [inline active/af4-trashman.py]: change task id to be the same as AF3's operator ID for symmetry's sake

**Review by WhiteAu — COMMENTED** [review]
change comment stuff.

**Pull request merged**